http/common_log.pony

```diff
@@ -22,10 +22,12 @@ class CommonLog is Logger
     list.push(" - ")
     list.push(_entry(request.url.user))
 
-    let time = PosixDate(Time.seconds()).format("%d/%b/%Y:%H:%M:%S +0000")
-    list.push(" [")
-    list.push(time)
-    list.push("] \"")
+    try // this one should never fail
```

Comment:
I won't hold up the PR on this account, but I don't think we can truly be confident in this not failing unless we have a test in the test suite that exercises this and proves that the format string works at least once. I didn't see such a test in the test suite - did I miss it?

Reply:
i couldnt find either

Reply:
i added a test for this.